### PR TITLE
python3Packages.jsonrpc{client,server}: init at {3.3.6,4.1.3}

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

Add new Python packages which are dependencies of the upcoming Electrum 4.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/jsonrpcserver/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  pname = "jsonrpcserver";
+  version = "4.1.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "649680c293facb6ae7c3f5c8028e4623c55195db5216847e9f25f85cba2d443a";
+  };
+
+  propagatedBuildInputs = [ apply_defaults jsonschema ];
+
```
Comment (jonringer) [inline pkgs/development/python-modules/jsonrpcserver/default.nix:18]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Comment (prusnak) [reply pkgs/development/python-modules/jsonrpcserver/default.nix:18]: Indeed the `tests` folder is not included in the release tarball. However, if I checkout from GitHub, the folder is there, but I still see `Ran 0 tests in 0.000s`, please advise.

This is the spec I am using:

```
{ stdenv
, buildPythonPackage
, fetchFromGitHub
}:

buildPythonPackage rec {
  pname = "apply_defaults";
  version = "0.1.4";

  src = fetchFromGitHub {
    owner = "bcb";
    repo = pname;
    rev = version;
    sha256 = "1nn2mch5spq8mhk741ia7zkb2bgjfn7bzxpavb476jzs38pb39d5";
  };

  meta = with stdenv.lib; {
    description = "Apply values to optional params";
    homepage = "https://github.com/bcb/apply_defaults";
    license = licenses.mit;
    maintainers = with maintainers; [ prusnak ];
  };

}
```

**Comment by prusnak** [comment]
Reworked the PR to checkout from GitHub (so the tests are included) and added `checkPhase = "pytest";` to each package.

**Comment by prusnak** [comment]
Not needed anymore. Electrum maintainers just removed these dependencies before the 4.0 release: https://github.com/spesmilo/electrum/commit/30f5be26acb182d3f8063c2f3afb09fc9f39433a

**Pull request closed without merging**